File: Python/números_Pares_impar/desafio.py
# Escreva um programa que receba um número inteiro do usuário e informe se ele é par ou ímpar.

while True:
    try:
        num = int(input('Digite um número inteiro é veja se ele e Impar ou Par: '))
    
        if num % 2 == 0:
            print(f'O número que você digitou foi {num} ele e PAR.')

        else:
            print(f'O número que você digitou é IMPAR.')

    except ValueError:
        print('\033[31mDigite um número inteiro valido! Tente novamente.\033[m')
        continue
    

File: JavaScript/Par-Impar/desafio.js
// Verifica se o número e PAR ou IMPAR

function Verificarnumero(num) {
    if (num % 2 === 0) {
        console.log(`O número ${num} é PAR.`)

    } else if (num % 2 === 1) {
        console.log(`O número ${num} é IMPAR.`)
    }
}

Verificarnumero(0)
